### ISSUE TITLE
Model size check doesn't seem to work

### ISSUE THREAD

**Issue by kleiti** [onnx/neural-compressor#39]
Even with my model that is less than 250KB in size, I get the onnx_data file after quantization. 

https://github.com/onnx/neural-compressor/blob/aabbf967cf7ea91c078c28c7b4dab043add5257b/onnx_neural_compressor/onnx_model.py#L245

**Comment by kleiti** [comment]
My model is just the .onnx file, no external weight file.

**Comment by mengniwang95** [comment]
Do you quantized the model with this API https://github.com/onnx/neural-compressor/blob/aabbf967cf7ea91c078c28c7b4dab043add5257b/onnx_neural_compressor/quantization/quantize.py#L28 ?
If so, during optimization it will save external data if model size > min_size: https://github.com/onnx/neural-compressor/blob/aabbf967cf7ea91c078c28c7b4dab043add5257b/onnx_neural_compressor/quantization/quantize.py#L43

**Comment by kleiti** [comment]
Yes, I do quantize using that API. Why is the size limit so low?

**Comment by mengniwang95** [comment]
Since onnxruntime can load external data automatically and we find model with external data format can be processed faster. But according to you case, the size limitation may need optimization.
Do you have some limitations to use external data format?
